.github/renovate.json

```diff
@@ -50,6 +50,7 @@
         "azure-policy/policy-kubernetes-addon-prod",
         "azure-policy/policy-kubernetes-webhook",
         "oss/kubernetes/kube-proxy",
+        "oss/v2/kubernetes/kube-proxy",
```

Comment:
where are the versions in components.json ? I only see `oss/kubernetes/kube-proxy` in there